<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowClass</class>
 <widget class="QWidget" name="ShowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="toProf">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #004080; /* Dark blue text color */
    background-color: transparent; /* Transparent background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 5px; /* Padding */
    margin: 5px; /* Margin */
}
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="prof_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}
</string>
   </property>
   <property name="text">
    <string>Professors</string>
   </property>
  </widget>
  <widget class="QLabel" name="class_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}
</string>
   </property>
   <property name="text">
    <string>Class</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>201</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-color: #E5F3FF; /* Lighter blue background for table */
    border: 1px solid #004080; /* Dark blue border */
    border-radius: 5px; /* Rounded corners */
}

QTableWidget::item {
    padding: 8px; /* Adjust padding as needed */
}

QHeaderView::section {
    background-color: #004080; /* Dark blue background for table header */
    color: #FFFFFF; /* White text color for table header */
    padding: 5px; /* Adjust padding as needed */
}
</string>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>90</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Student</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Grade</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="Time_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}
</string>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="toTime">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #004080; /* Dark blue text color */
    background-color: transparent; /* Transparent background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 5px; /* Padding */
    margin: 5px; /* Margin */
}
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="toLecture">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #004080; /* Dark blue text color */
    background-color: transparent; /* Transparent background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 5px; /* Padding */
    margin: 5px; /* Margin */
}
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="toClass">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #004080; /* Dark blue text color */
    background-color: transparent; /* Transparent background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 5px; /* Padding */
    margin: 5px; /* Margin */
}
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="background_image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Lecture_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}
</string>
   </property>
   <property name="text">
    <string>Lecture Hall</string>
   </property>
  </widget>
  <zorder>background_image</zorder>
  <zorder>toProf</zorder>
  <zorder>prof_label</zorder>
  <zorder>class_label</zorder>
  <zorder>tableWidget</zorder>
  <zorder>Time_label</zorder>
  <zorder>ok_button</zorder>
  <zorder>toTime</zorder>
  <zorder>toLecture</zorder>
  <zorder>toClass</zorder>
  <zorder>Lecture_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
